components/dropbox/actions/download-file-to-tmp/download-file-to-tmp.mjs

```diff
@@ -0,0 +1,46 @@
+import dropbox from "../../dropbox.app.mjs";
+import common from "../common/common.mjs";
+import fs from "fs";
+import { file } from "tmp-promise";
+
+export default {
+  ...common,
+  name: "Download File To TMP",
+  description: "Download a specific file to the temporary directory. [See docs here](https://dropbox.github.io/dropbox-sdk-js/Dropbox.html#filesDownload__anchor)",
```

Comment:
```suggestion
  description: "Download a specific file to the temporary directory. [See the documentation](https://dropbox.github.io/dropbox-sdk-js/Dropbox.html#filesDownload__anchor)."
```

Reply:
Changed "docs here" to "the documentation" to follow Google's Developer docs style guide

Reply:
Please address this issue, but don't commit the suggestion it's missing the trailing comma